README.md

```diff
@@ -41,6 +41,24 @@ MineColonies crashes every time? Have a suggestion? Found a bug? Create an issue
 
 ## For Developers
 
+### Adding MineColonies as Maven Dependecy
+
+In Forge, add the following to your `build.gradle`:
+
+```gradle
+repositories {
+    	maven { url 'https://ldtteam.jfrog.io/ldtteam/modding' }
+}
+
+dependencies {
+    	implementation fg.deobf("com.ldtteam:minecolonies:${minecolonies_version}")

```

Comment:
Please fix the indent

Reply:
> Please fix the indent

I changed it to 4 spaces now but if you meant another indent please let me know